### PR TITLE
v1.11 backports 2022-01-26

### PR DESCRIPTION
**Pull request by kkourt**
* #18483 -- node: Don't skip masquerading for External node IPs (@pchaigno)
  * There was a conflict: resolved according to @pchaigno instructions
 * #18538 -- contrib: Fix release script helm value generation (@joestringer)
 * #18546 -- docs: Document required kernel configuration options (@pchaigno)
 * #18499 -- daemon: Fix missing errors in KPR init (@brb)
 * #18484 -- bpf: Move to-stack trace after host firewall (@YutaroHayakawa)
 * #18479 -- monitor: Output non-trace messages to stderr (@YutaroHayakawa)
 * #18592 -- daemon: Allow to enable PCAP recorder in non-lb mode (@brb)
 * #18564 -- route: sort by priority to identify the default one (@jibi)
 * #18606 -- byteorder: use native instructions in host/network order conversion (@tklauser)
 * #18582 -- node: Fix bug where node ipsets are never cleaned (@pchaigno)
 * #18554 -- Enable CI for feature branches (@jibi)
    * Had conflicts: followed @jibi's 1.10 backport PR: https://github.com/cilium/cilium/pull/18617
 * #18553 -- metrics: Expose xfrm stats in prometheus (@sayboras)
     * Minor conflict in `CODEOWNERS`
 * #18469 -- Ignore empty Maglev map from breaking `bpf lb maglev list` (@ti-mo)

Once this PR is merged, you can update the PR labels via:
```upstream-prs
$ for pr in 18483 18538 18546 18499 18484 18479 18592 18564 18606 18582 18554 18553 18469; do contrib/backporting/set-labels.py $pr done 1.11; done
```

## Conflicts

 * #18554 -- Enable CI for feature branches (@jibi)
    * Had conflicts: followed @jibi's 1.10 backport PR: https://github.com/cilium/cilium/pull/18617
 * #18483 -- node: Don't skip masquerading for External node IPs (@pchaigno)
     * There was a conflict: resolved according to @pchaigno instructions
  * #18553 -- metrics: Expose xfrm stats in prometheus (@sayboras)
     * Minor conflict in `CODEOWNERS`

### CODE REVIEW

**Comment by kkourt** [comment]
/test-backport-1.11

**Review by pchaigno — APPROVED** [review]
My changes look good :+1: Thanks Kornilios!

**Review by tklauser — APPROVED** [review]
Thanks, looks good for my change.

**Review by brb — APPROVED** [review]
My changes LGTM, thanks.

**Review by jibi — APPROVED** [review]
:heavy_check_mark:  for my changes, thanks!

File: CODEOWNERS [hunk 1]
```diff
@@ -14,6 +14,7 @@
 /pkg/byteorder/ @cilium/bpf @cilium/api
 /pkg/client @cilium/api
 /pkg/k8s/apis/cilium.io/v2/ @cilium/api
+/pkg/datapath/linux/ipsec/xfrm_collector* @cilium/metrics
```
Comment (sayboras) [inline CODEOWNERS:17]: nit: seems like the original list is following alphabetical order.

**Review by sayboras — APPROVED** [review]
Thanks, looks good for my commits :100:

**Comment by ti-mo** [comment]
Thanks Kornilios, looks good!

**Review by ti-mo — APPROVED** [review]

**Comment by YutaroHayakawa** [comment]
Thanks Kornilios my change looks good to me!

**Review by YutaroHayakawa — APPROVED** [review]

**Review by joestringer — APPROVED** [review]

**Comment by glibsm** [comment]
/test-backport-1.11

**Comment by glibsm** [comment]
/test-backport-1.11

**Review by glibsm — APPROVED** [review]

**Pull request merged**